lib/init.coffee

```diff
@@ -6,6 +6,14 @@ module.exports =
     rcFile:
       type: 'string'
       default: ''
+    showCodes:
+      description: "Show pylint codes? (E1101, etc.)"
+      type: 'boolean'
+      default: false
+    showReadableCodes:
+      description: "Show the short human-readable version of pylint codes? (e.g., unused-import for W0611)"
```

Comment:
Consider instead something like:

title: "Show Pylint Symbols"
description: "Displays the human-readable Pylint symbol in the Linter message  (e.g., unused-import, etc.)"